src/prerequestscript.js

```diff
@@ -497,17 +546,16 @@ var AcquiaHttpHmac = function () {
 
 var method = request.method.toUpperCase(),
     path = request.url,
-    signed_headers = {};
+    signed_headers = {},
+    content_type = "application/json",
+    body = request.data;
     
-var sign_parameters = {request, method, path, signed_headers};
+var sign_parameters = {request, method, path, signed_headers, content_type, body};
```

Comment:
The default content type falls back to `application/json` so it doesn't need passed in here. Suggest removal for version 1.1.5. Ah, actually, I think `content_type` isn't used at all. Perhaps double check or maybe this is old code from merge/refactoring.

Reply:
I have to double check but I'm pretty sure one of the API's require the content_type be passed otherwise it rejects the request

Reply:
Yeah, I was wrong. I just spotted my mistake whilst code reviewing my 1.1.5 changes. I am retracting this comment.